pkg/apis/serving/v1alpha1/revision_validation.go

```diff
@@ -170,13 +170,10 @@ func validatePercentageAnnotationKey(annotations map[string]string, resourcePerc
 	}
 	value, err := strconv.ParseFloat(v, 32)
 	if err != nil {
-		return &apis.FieldError{
-			Message: fmt.Sprintf("Invalid value %s for annotation %s: %v", v, resourcePercentageAnnotationKey, err),
-			Paths:   []string{resourcePercentageAnnotationKey},
-		}
+		return apis.ErrInvalidValue(v, resourcePercentageAnnotationKey)
```

Comment:
I don't think this will show up properly.  We want it in `[ ... ]` and the second arg won't do that.  See my prior example for what I think you need here.